x-pack/plugin/core/src/main/resources/fleet-files-ilm-policy.json

```diff
@@ -0,0 +1,25 @@
+{
+  "phases": {
+    "hot": {
+      "min_age": "0ms",
+      "actions": {
+        "rollover": {
+          "max_size": "10gb",
+          "max_age": "30d"
```

Comment:
[Question]
I'm not fully aware of the implementation of the file upload feature, so I might be asking about things that were decided already, but let me ask to understand this PR better.
 
I'm assuming that `fleet-files` is for metadata and `fleet-file-data` is the actual file. If so, why aren't `max_age` and `delete.min_age` the same for both indexes?

Reply:
Hi @AndersonQ this is a good question, I put these in as a starter and forgot to raise it with the team what the values should be, I've asked the question and will verify what values we want. https://github.com/elastic/kibana/issues/143459#issuecomment-1335298224

Reply:
I added @pzl and @michel-laterman as reviewers here as the two people I know are working on file upload related features in Fleet server.

Reply:
OK so file data will be deleted after 7 days and metadata after 90. We don't want to keep the files around for long because I believe in the endpoint use case they could be malicious samples etc. Retaining the metadata for longer metadata will keep a record that the file did exist.